packages/xo-server/src/api/vm.mjs

```diff
@@ -1327,6 +1329,84 @@ importFromEsxi.params = {
   vm: { type: 'string' },
 }
 
+export async function importMultipleFromEsxi({
+  concurrency = 2,
+  host,
+  network,
+  password,
+  sr,
+  sslVerify = true,
+  stopSource = false,
+  stopOnError = false,
+  thin = false,
+  user,
+  vms,
+}) {
+  const task = await this.tasks.create({ name: `importing vms ${vms.join(',')}` })
+  Task.set('total', vms.length)
+  Task.set('done', 0)
+  Task.set('progress', 0)
+  let done = 0
+  return task.run(async () => {
+    const result = {}
+    try {
+      await asyncEach(
+        vms,
+        async vm => {
+          await new Task({ name: `importing vm ${vm}` }).run(async () => {
+            const vmUuid = await this.migrationfromEsxi({
+              host,
+              user,
+              password,
+              sslVerify,
+              thin,
+              vm,
+              sr,
+              network,
+              stopSource,
+            })
+            result[vm] = vmUuid
+          })
+          done++
+          Task.set('done', done)
+          Task.set('progress', Math.round((done * 100) / vms.length))
+        },
+        {
+          concurrency,
+          stopOnError,
+        }
+      )
+      return result
+      // error is an AggregatedError containing all the errors if stopOnError is true, the first error otherwise
```

Comment:
I believe it's the opposite.

Also, it would make more sense to move this comment in the `catch` block.